packages/typespec-ts/test/modularIntegration/clientStructureDefault.spec.ts

```diff
@@ -29,7 +29,7 @@ describe("Client Structure Default Rest Client", () => {
 
   it("should call operation three correctly", async () => {
     try {
-      const result = await client.three();
+      const result = await client.foo.three();
```

Comment:
This should be expected. as by default, we should enable hierarchy client.

Reply:
does that mean our earlier code have a bug for this case?

Reply:
no, we disable it before and we don't treat interface as hierarchy if there's no `@operationGroup` decorator on it.